app/resources/home.html.template

```diff
@@ -1,13 +1,83 @@
-{# Generic, default home page for a new site.  Fill in content below.       #}
-
 {% extends "static-base.html.template" %}
+{% load i18n %}
 
 {% block content %}
-  <h1>Welcome</h1>
 
-  <p>You are now running Person Finder.
+  {% if config.brand == "google" %}
+
+    <h1>
+      Google Person Finder helps people reconnect with friends and loved ones
+      in the aftermath of natural and humanitarian disasters.
+    </h1>
+
+    <div class="storyboard">
+      <div class="storyboard-section">
+        <table>
+          <tr>
+            <td>
+              <img src="{{env.global_url}}/separated-81x80.png" width="81" height="80">
+            </td>
+            <td>
+              <img src="{{env.global_url}}/search-100x80.png" width="100" height="80">
+            </td>
+          </tr>
+          <tr class="captions">
+            <td>A crisis strikes and people get separated.</td>
+            <td>They let the world know they are looking for someone.</td>
```

Comment:
What are these captions? can you provide a screenshot?
How about translations?
Also, the text is confusing to me (maybe I don't have enough context)